stylesheets/components/alert/alert.css

```diff
@@ -0,0 +1,96 @@
+/* Default alert */
+.alert {
+  align-items: center;
+  border-radius: var(--radius-sm);
+  display: flex;
+  font-size: calc(var(--size-9) + (1.6 * var(--size-1))); /* 1.1rem */
+  font-weight: var(--weight-medium);
+  justify-content: flex-start;

```

Comment:
Just a suggestion, you may want to write ```display: flex``` and allied properties right below each other. This'll improve readability and will ease debugging if needed.

Everything else looks good to me.

Reply:
Thanks for the suggestion👍. 
But I'm trying to arrange the properties in alphabetical order.